eve/main.yml

```diff
@@ -2671,3 +2693,165 @@ stages:
+      type: local
+    steps:
+      - ShellCommand: *add_final_status_artifact_failed
+      - SetProperty: *set_premerge_artifacts_name
+      - SetProperty: *set_premerge_url
+      - TriggerStages:
+          stage_names:
+            - publish-shell-ui
+          haltOnFailure: true
+      - ShellCommand: *add_final_status_artifact_success
+      - Upload: *upload_final_status_artifact
+
+  publish-shell-ui:
+    worker: *build_worker
+    steps:
+      - ShellCommand: *wait_for_docker
+      - ShellCommand: *image_registry_login
+      - SetPropertyFromCommand: *set_version_prop
+      - SetPropertyFromCommand:
+          name: Set is_latest_dev property
+          property: is_latest_dev
+          command: |
+            if [ "$IS_LATEST_DEV" != unset ]; then
+                echo "$IS_LATEST_DEV"
+            elif [ "$IS_LATEST_STABLE" = true ]; then
+                echo false
+            else
+                LATEST_DEV_BRANCH=$(
+                    git ls-remote --heads "$GIT_REFERENCE" | \
+                    awk -F/ '$3 == "development" { print $4 }' | sort -V | tail -n 1
+                )
+                [ "$GIT_BRANCH" = "development/$LATEST_DEV_BRANCH" ] && echo true || echo false
+            fi
+          env:
+            GIT_REFERENCE: '%(prop:git_reference)s'
+            GIT_BRANCH: '%(prop:branch)s'
+            IS_LATEST_DEV: '%(prop:is_latest_dev:-unset)s'
+            IS_LATEST_STABLE: '%(prop:is_latest_stable:-unset)s'
+      - SetPropertyFromCommand:
+          name: Set is_stable_version property
+          property: is_latest_stable
+          command: |
+            if [ "$IS_LATEST_STABLE" != unset ]; then
+                echo "$IS_LATEST_STABLE"
+            elif [ "$IS_LATEST_DEV" = true ]; then
+                echo false
+            else
+                LATEST_STABLE_RELEASE=$(
+                    git ls-remote --tags --refs "$GIT_REFERENCE" | \
+                    awk -F/ '$3 ~ /^[0-9]+\.[0-9]+\.[0-9]+$/ { print $3 }' | sort -V | tail -n 1
+                )
+                [ "$VERSION" = "$LATEST_STABLE_RELEASE" ] && echo true || echo false
+            fi
+          env:
+            GIT_REFERENCE: '%(prop:git_reference)s'
+            IS_LATEST_DEV: '%(prop:is_latest_dev)s'
+            IS_LATEST_STABLE: '%(prop:is_latest_stable:-unset)s'
+            VERSION: '%(prop:metalk8s_version)s'
+      - SetPropertyFromCommand:
+          name: Set image_registry_url property
+          property: image_registry_url
+          command: |
+            if [ "$USE_PRODUCTION_REGISTRY" = true ]; then
+                PROJECT='%(secret:harbor_prod_project)s'
+            else
+                PROJECT='%(secret:harbor_dev_project)s'
+            fi
+            echo "$REGISTRY_HOST/$PROJECT"
+          env: *_env_image_registry
+      - SetPropertyFromCommand:
+          name: Set is_dev property
+          property: is_dev
+          value: false
+          doStepIf: '%(prop:use_production_registry:-false)s'
```

Comment:
This should be a `SetProperty` (not `SetPropertyFromCommand`)

Reply:
Yup, I saw it, it's fixed.